src/components/overview/LeftPanel.tsx

```diff
@@ -69,95 +89,89 @@ export default function LeftPanel({ onOpenSettingsModal }: Props) {
-                        status: 'error',
-                        duration: 2000,
-                        isClosable: true
-                      })
-                    })
-                    return !value
-                  })
-                }}
+
+          <SearchBar />
+          <Divider />
+        </VStack>
+        <VStack
+          w="full"
+          h="full"
+          overflowY="auto"
+          overflowX="hidden"
+          ref={parent}
+          px={4}
+          spacing={1}
+          sx={useScrollbarStyles()}
+        >
+          {searchValue.length === 0 &&
+            collections.map((collection: Collection, index: number) => (
+              <CollectionButton
+                collection={collection}
+                index={index}
+                key={`${collection.name} + ${index}`}
+                isFocused={index == selection.collectionIndex}
+                isMacroOutputEnabled={isMacroOutputEnabled}
+                setFocus={(index) => changeSelectedCollectionIndex(index)}
+                toggleCollection={() =>
+                  onCollectionUpdate(
+                    {
+                      ...collections[index],
+                      active: !collections[index].active
+                    },
+                    index
+                  )
+                }
               />
-            </Box>
-          </Tooltip>
-        </HStack>
-        <Divider />
+            ))}
+          {searchValue.length === 0 && (
+            <Button
+              rounded={borderRadiusStandard}
+              size="md"
+              w="full"
+              variant="yellowGradient"
+              p={1}
+              minH="2.5rem"
+              margin={1}
+              leftIcon={<AddIcon />}
+              fontSize="md"
+              onClick={onNewCollectionButtonPress}
+            >
+              New Collection
+            </Button>
+          )}
+        </VStack>
+      </VStack>
+      <VStack w="full" px={4} pb={4}>
         <Button
-          size="lg"
           w="full"
-          variant="yellowGradient"
-          p={2}
-          leftIcon={<AddIcon />}
-          fontSize="md"
-          onClick={onNewCollectionButtonPress}
+          colorScheme={isMacroOutputEnabled ? 'green' : 'orange'}
+          size="sm"
+          onClick={() => {
+            updateMacroOutput(isMacroOutputEnabled).catch((e) => {
+              error(e)
+              toast({
+                title: `Error ${
+                  isMacroOutputEnabled? 'disabling' : 'enabling'
+                } macro output`,
+                description: `Unable to ${
+                  isMacroOutputEnabled? 'disable' : 'enable'
+                } macro output, please re-open the app. If that does not work, please contact us on Discord.`,
+                status: 'error',
+                duration: 2000,
+                isClosable: true
+              })
+            })
+            changeMacroOutputEnabled(isMacroOutputEnabled)
```

Comment:
Should it be updating the `changeMacroOutputEnabled` if it gets an error running `updateMacroOutput`?

Reply:
Tweaked: [008d6a3](https://github.com/WootingKb/wooting-macros/pull/194/commits/008d6a318895a345bfc178ce8ea6b1e43220b6b9)